Corrected readme for IBM Db2 (#13344)
DataDog/integrations-core: ibm_db2/README.md

On Linux there may be need for XML functionality. If you encounter errors during
the build process, install `libxslt-dev` (or `libxslt-devel` for RPM).

#### Enable monitoring

> To monitor an instance, its associated databases, and database objects according to various health indicators and turn on some default database system monitor switches (snapshot monitor's statement, lock, table and buffer pool) 

Reviewer: ```suggestion
To monitor the health of an instance, its associated databases, and database objects, enable the database system monitor switches for each of the objects you want to monitor: 
* statement
* lock
* tables
* buffer pool
```

On Linux there may be need for XML functionality. If you encounter errors during
the build process, install `libxslt-dev` (or `libxslt-devel` for RPM).

#### Enable monitoring

To monitor the health of an instance, its associated databases, and database objects, enable the database system monitor switches for each of the objects you want to monitor: 
* statement
* lock
* tables

> * buffer pool

Reviewer 2: Can we capitalize these bullets?

Author: Done. Thanks for the review
